src/test/java/io/gingersnapproject/testcontainers/hotrod/CacheManagerContainer.java

```diff
@@ -0,0 +1,62 @@
+package io.gingersnapproject.testcontainers.hotrod;
+
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.Base58;
+import org.testcontainers.utility.DockerImageName;
+
+public class CacheManagerContainer extends HotRodContainer<CacheManagerContainer> {
+
+   private final String dbKind;
+   private String databaseUrl;
+   private String dbUser;
+   private String dbPassword;
+   private String[] rules;
+
+   public CacheManagerContainer(String kind) {
+      super(DockerImageName.parse(String.format("cache-manager-%s:latest", kind)));
+      this.dbKind = kind;
+      withNetworkAliases("infinispan-" + Base58.randomString(6));
+      withExposedPorts(8080, 11222);
+      waitingFor(Wait.forHttp("/q/health").forPort(8080));
+   }
+
+   public CacheManagerContainer withDatabaseUrl(String databaseUrl) {
+      this.databaseUrl = databaseUrl;
+      return self();
+   }
+
+   public CacheManagerContainer withDatabaseUser(String dbUser) {
+      this.dbUser = dbUser;
+      return self();
+   }
+
+   public CacheManagerContainer withDatabasePassword(String dbPassword) {
+      this.dbPassword = dbPassword;
+      return self();
+   }
+
+   public CacheManagerContainer withRules(String ... rules) {
+      this.rules = rules;
+      return self();
+   }
+
+   @Override
+   public void start() {
+      withEnv(String.format("quarkus_datasource_%s_username", dbKind), dbUser);
+      withEnv(String.format("quarkus_datasource_%s_password", dbKind), dbPassword);
+      withEnv(String.format("quarkus_datasource_mysql_reactive_url", dbKind), "");
```

Comment:
I think just changing this property in the application.properties to be for dev to be fine. Then this line isn't needed.